tests/unit/test_is_stuck.py

```diff
@@ -75,12 +75,12 @@ def _impl_syntax_error_events(
     def _impl_unterminated_string_error_events(
         self, event_stream: EventStream, random_line: bool
     ):
-        for _ in range(4):
+        for i in range(4):
             ipython_action = IPythonRunCellAction(code='print("hello')
             event_stream.add_event(ipython_action, EventSource.AGENT)
-            line_number = str(random.randint(1, 10)) if random_line else '1'
+            line_number = i * 10 if random_line else '1'
```

Comment:
I think alternatively, we could fix a seed for the random num generator, but I am not familiar with the context here so I'll leave the decision to you.

Reply:
Thanks!
The random wasn't a really needed, I just needed different numbers between actions, so here the counter works well enough. :)